tsconfig.json

```diff
@@ -1,23 +1,18 @@
 {
     "compilerOptions": {
-        "lib": ["ES2021"],
+        "target": "ESNext",
+        "module": "CommonJS",
         "outDir": "./dist",
-        "target": "ES2021",
+        "lib": ["ES2021"],
+
         "declaration": true,
-        "module": "CommonJS",
         "noImplicitAny": true,
         "removeComments": false,
         "strictNullChecks": true,
         "resolveJsonModule": true,
         "moduleResolution": "Node",
         "preserveConstEnums": true,
-        "declarationDir": "./dist/src/types"
+        "useDefineForClassFields": false
```

Comment:
Why is this option here?

Reply:
quick fixes ts error 2612 on build

```
src/structures/User.ts(203,21): error TS2612: Property 'id' will overwrite the base property in 'BaseUser'. If this is intentional, add an initializer. Otherwise, add a 'declare' modifier or remove the redundant declaration.

Error: error occured in dts build
    at Worker.<anonymous> (D:\code\PteroJS\node_modules\tsup\dist\index.js:2157:26)
    at Worker.emit (node:events:527:28)
    at MessagePort.<anonymous> (node:internal/worker:232:53)
    at MessagePort.[nodejs.internal.kHybridDispatch] (node:internal/event_target:643:20)
    at MessagePort.exports.emitMessage (node:internal/per_context/messageport:23:28)
Error: Failed to compile. Check the logs above.
    at error (D:\code\PteroJS\node_modules\rollup\dist\shared\rollup.js:198:30)
    at throwPluginError (D:\code\PteroJS\node_modules\rollup\dist\shared\rollup.js:21718:12)
    at Object.error (D:\code\PteroJS\node_modules\rollup\dist\shared\rollup.js:22672:20)
    at Object.error (D:\code\PteroJS\node_modules\rollup\dist\shared\rollup.js:21895:42)
    at Object.transform (D:\code\PteroJS\node_modules\tsup\dist\rollup.js:7217:20)
    at D:\code\PteroJS\node_modules\rollup\dist\shared\rollup.js:22879:40
DTS Build error
```

Reply:
compiler is moaning about overwriting the base prop and would like an initializer if intentional 
https://github.com/PteroPackages/PteroJS/blob/c472cbcf35c47f45ca5926b5150b8a1a765e3596/src/structures/User.ts#L199

Reply:
Ah right, I'll sort that out in another commit.